app/controllers/publishers_controller.rb

```diff
@@ -423,6 +423,43 @@ def balance
     end
   end
 
+  def banner_editor_data
+    # Handle edge case where user hasn't created channels yet.
+    no_channels_banner = current_publisher.site_banners.find_by(channel_id: "00000000-0000-0000-0000-000000000000")
```

Comment:
Would it be possible to get rid of lines 427 to 431?

Reply:
Done ✔️